pymesync.py

```diff
@@ -133,6 +134,15 @@ def create_time(self, parameter_dict):
         ``parameter_dict`` is a python dictionary containing the time
         information to send to TimeSync.
         """
+        if type(parameter_dict['duration']) is not int:
```

Comment:
This [SO post](http://stackoverflow.com/a/1549854) suggests that if you have to check type, you should use ``isinstance`` since it accounts for subclasses too. The only reason this would affect anything is if someone somehow passed in a subclass of ``int``, but I do use ``isinstance`` on [line 629](https://github.com/osuosl/pymesync/blob/develop/pymesync.py#L629) so you may want to modify this for consistency.